diff --git a/analysis/optuna_studies.db b/analysis/optuna_studies.db
index b0432eb..848e216 100644
Binary files a/analysis/optuna_studies.db and b/analysis/optuna_studies.db differ
diff --git a/wandb/latest-run b/wandb/latest-run
index bc957aa..212c0a1 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20251226_161318-r133h7t1
\ No newline at end of file
+run-20251226_165328-71wjwnpz
\ No newline at end of file
